Dockerfile

```diff
@@ -70,16 +95,15 @@ EXPOSE 3000
 CMD ["bundle", "exec", "puma"]
 
 # ------------------------------------------------------------------------------
-# test stage
+# test
 # ------------------------------------------------------------------------------
+FROM web as test
 
-FROM release as test
-
-RUN apt-get install -yy firefox-esr
+RUN apt-get install -qq -y firefox-esr
 
 ARG gecko_driver_version=0.26.0
 
-RUN wget https://github.com/mozilla/geckodriver/releases/download/v$gecko_driver_version/geckodriver-v$gecko_driver_version-linux64.tar.gz
+RUN wget --quiet  https://github.com/mozilla/geckodriver/releases/download/v$gecko_driver_version/geckodriver-v$gecko_driver_version-linux64.tar.gz
```

Comment:
Is there a reason you chose to make the output as quiet as possible? There can be lots of output but the trade off is that with no output you can't see what progress the build command is making, which can be frustrating when it takes a while to complete.

Reply:
wget is very verbose, that was the reason, but I think your right that more information at this point is better.